README.md

```diff
@@ -1,41 +1,162 @@
-# Servicer
-CI/CD Automation Framework
+# Servicer #
 
-The primary goal of this framework is to completely automate creating, updating, and removing an environment (a collection of services). To accomplish this, this framework executes in your CI environment (eg: Bitbucket Pipelines, Gitlab, Jenkins, CircleCI, etc) and interfaces directly with cloud infrastructure provider libraries (AWS, Google Cloud, Azure, etc).
+Servicer is an independent CI framework that can be concurrently run on the laptops used by your developers, and/or concurrently on top of major CI platforms, like Bitbucket Pipelines, Jenkins, and CircleCI.
```

Comment:
I can see the opportunity for misunderstanding here. It can be used 'concurrently' if the repo is kept as the single source of truth, and any release/deployment steps are preceded with Servicer verifying that the local branch is up to date with the branch on origin.